android/src/main/java/com/reactnativestripesdk/CardFormViewManager.kt

```diff
@@ -0,0 +1,62 @@
+package com.reactnativestripesdk
+
+import com.facebook.react.bridge.ReadableArray
+import com.facebook.react.bridge.ReadableMap
+import com.facebook.react.common.MapBuilder
+import com.facebook.react.uimanager.ReactStylesDiffMap
+import com.facebook.react.uimanager.SimpleViewManager
+import com.facebook.react.uimanager.ThemedReactContext
+import com.facebook.react.uimanager.annotations.ReactProp
+
+class CardFormViewManager : SimpleViewManager<CardFormView>() {
+  override fun getName() = "CardForm"
+
+  private var cardViewInstanceMap: MutableMap<String, Any?> = mutableMapOf()
+
+  override fun getExportedCustomDirectEventTypeConstants(): MutableMap<String, Any> {
+    return MapBuilder.of(
+      CardFocusEvent.EVENT_NAME, MapBuilder.of("registrationName", "onFocusChange"),
+      CardFormCompleteEvent.EVENT_NAME, MapBuilder.of("registrationName", "onCardComplete"))
+  }
+
+  override fun receiveCommand(root: CardFormView, commandId: String?, args: ReadableArray?) {
+    when (commandId) {
+      "focus" -> root.requestFocusFromJS()
+      "blur" -> root.requestBlurFromJS()
+      "clear" -> root.requestClearFromJS()
+    }
+  }
+
+  @ReactProp(name = "dangerouslyGetFullCardDetails")
+  fun setDangerouslyGetFullCardDetails(view: CardFormView, dangerouslyGetFullCardDetails: Boolean = false) {
+    view.setDangerouslyGetFullCardDetails(dangerouslyGetFullCardDetails);
+  }
+
+  @ReactProp(name = "postalCodeEnabled")
+  fun setPostalCodeEnabled(view: CardFormView, postalCodeEnabled: Boolean = false) {
+    view.setPostalCodeEnabled(postalCodeEnabled);
+  }
+
+  @ReactProp(name = "placeholder")
+  fun setPlaceHolders(view: CardFormView, placeholder: ReadableMap) {
+    view.setPlaceHolders(placeholder);
+  }
```

Comment:
These are outdated, no? Or do they work?

Reply:
it works on android apart from `card field` placeholder

Reply:
@thorsten-stripe so please decide you we want to get rid of this until fully working

Reply:
Yes, as discussed please remove everything that currently doesn't work across platforms.